converters/toZigbee.js

```diff
@@ -35,6 +35,12 @@ const options = {
     },
 };
 
+const siterwellGs361SystemModes = {
```

Comment:
Is this mapping different than the system_mode mapping for other thermostats?

Reply:
yes. Take a look. The GS361 has only 3 modes and the enumeration is also different.